update.sh

```diff
@@ -17,26 +17,26 @@ function usage() {
     - update.sh 8 buster-slim,buster # Update only buster's slim and buster variants for version 8
     - update.sh -s 8 stretch         # Update only stretch variant for version 8, skip updating Alpine and Yarn
     - update.sh . alpine             # Update the alpine variant for all versions
-    - update.sh -t                   # Update .travis.yml only
+    - update.sh -b                   # Update CI files only
 
   OPTIONS:
     -s Security update; skip updating the yarn and alpine versions.
-    -t Travis CI config update only
+    -t CI config update only
```

Comment:
`t` or `b`?

Reply:
Seems like it's still `t` based on line 38, just the change on line 20 that inconsistent then?

Reply:
Yeah, I tried changing it to `c` at one point and it just stopped working properly, so I just undid the letter change

Reply:
OK, I figured out my issue with not updating the `getopts`, so it's "b" now